src/cascadia/TerminalSettingsModel/Command.cpp

```diff
@@ -303,7 +306,8 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             // should also be used for unbinding.
 
             // create an "invalid" ActionAndArgs
-            result->_action = make<ActionAndArgs>();
+            result->_ActionAndArgs = make<implementation::ActionAndArgs>();
+            result->_nestedCommand = true;

```

Comment:
Oh this one right here is the reason. Maybe in the doc comment for `IsNestedCommand` we should call this out.